src/components/EditPen.vue

```diff
@@ -36,5 +38,9 @@ export default class EditPen extends Vue {
 			transform: scaleX(-1);
 		}
 	}
+
+	&__edit-action-text {
+		@include sr-only();
```

Comment:
https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_screen-reader.scss#L6 Cool :)

Reply:
My IDE allows me to click that in `src/styles/_main.scss`